packages/orbit-components/config/fetchIcons.mts

```diff
@@ -38,6 +38,10 @@ interface FigmaComponents {
   i18n: null;
 }
 
+interface Images {
```

Comment:
Just to clarify, any reason for being interface and not type?

Reply:
we decided to enforce interfaces, so I will keep that :)